### PR TITLE
Azure Deployment

### PR DESCRIPTION
**Pull request by nelsonic**
This PR is _probably_ not going to pass CI (_for now_) ...
Documented _all_ the steps in: https://github.com/dwyl/learn-phoenix-framework/blob/master/production-deployment.md
Just _implementing_ them here. 👍

### CODE REVIEW

**Comment by iteles** [comment]
@nelsonic Let me know when you need a review of this 👍 💃

**Comment by nelsonic** [comment]
HI @iteles  & @katbow,
the "setup" code for deploying to staging is in this PR (_along with a `master` rebase which was necessary to fix merge conflicts ..._)
Please review/comment/merge when you can.
I will open a `new` PR with instructions to ship to production. 👍

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Healthlocker [![Build Status](https://travis-ci.org/healthlocker/healthlocker.svg?branch=master)](https://travis-ci.org/healthlocker/healthlocker) [![Deps Status](https://beta.hexfaktor.org/badge/all/github/healthlocker/healthlocker.svg)](https://beta.hexfaktor.org/github/healthlocker/healthlocker)
 
+<<<<<<< HEAD
```
Comment (katbow) [inline README.md]: Missed from merge conflict

File: README.md [hunk 2]
```diff
@@ -9,6 +10,22 @@ communication. Clinicians can also stay updated on goals and coping strategies
 created by their SUs in Healthlocker, which keeps them up-to-date with what is
 important to their SU. A full list of features can be found
 [here](https://github.com/healthlocker/healthlocker#current-features).
+=======
```
Comment (katbow) [inline README.md]: here too!

File: README.md [hunk 3]
```diff
@@ -9,6 +10,22 @@ communication. Clinicians can also stay updated on goals and coping strategies
...
+## _Why_?
+
+Healthlocker's mission is to enable people to
```
Comment (katbow) [inline README.md]: Will this be for @MollieCourtenay to add to as well?

**Pull request merged**